v3/internal/runtime/desktop/@wailsio/runtime/src/android.ts

```
@@ -0,0 +1,87 @@
+    export function SetPreviewEnabled(enabled: boolean): Promise<void> {
+        return call(LinksSetPreviewEnabled, { enabled });
+    }
+}
+
+export namespace UserAgent {
+    export function SetUserAgent(ua: string): Promise<void> {
+        return call(UserAgentSet, { ua });
+    }
+}
+
+export const Set = UserAgent.SetUserAgent;
```

Biome (2.3.14)

[error] Do not shadow the global "Set" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)